CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.1.7] - 2022-07-06
+
+### Fixed
+- Change was made on helm_release_ingress_nginx to start it without depending on prometheus.
```

Comment:
I would put: 

Add default values for `controller.metrics.enabled` and `controller.metrics.serviceMonitor.enabled` to ingress-nginx Helm release in order to allow deploying it without having Prometheus as a dependency.

@Ciejo agree?

Reply:
Remember setting the ` characters at the start and end of controller.metrics.enabled and controller.metrics.serviceMonitor.enabled in CHANGELOG.md when writing the change details to get them nicely formatted.